packages/jaspr/lib/src/browser/options.dart

```diff
@@ -0,0 +1,52 @@
+import 'dart:async';
+
+import '../../jaspr.dart';
+
+/// Main class for initializing the Jaspr framework on the client.
+///
+/// Call [Jaspr.initializeApp()] at the start of your app, before any calls to [runApp].
+class Jaspr {
```

Comment:
I'd make this `abstract final` to ensure no one thinks it's meant to be initialized.

```suggestion
abstract final class Jaspr {
```